kit/build.gradle

```diff
@@ -25,3 +28,10 @@ uploadArchives {
         }
     }
 }
+
+def installer = install.repositories.mavenInstaller
+def deployer = uploadArchives.repositories.mavenDeployer
+
+[installer, deployer]*.pom*.whenConfigured { pom ->
+    pom.dependencies.findAll{ it.groupId == 'org.openjfx' }.collect { it.classifier = null }
+}
```

Comment:
What are these last few lines meant to do?

Reply:
It removes the classifier from the javafx dependencies when the maven pom file for scene builder is generated. The classifier defines the platform specific part, so removing it will cause the pom to only contain platform independent javafx dependencies.

Reply:
That makes sense. Thanks for the explanation :)